images/scripts/arch.setup

```diff
@@ -112,5 +112,11 @@ pacman -Scc --noconfirm
 # Reduce image size, remove cloud-init, reflector
 pacman -Rscn --noconfirm cloud-init reflector
 
+# Remove the no longer required systemd units as they are not required and seem
+# to cause an issue delaying sshd to be started after network online target
+# timeout. As reflector-init is reverse dependency on sshd which needs network-online.target
+rm /etc/systemd/system/reflector-init.service
```

Comment:
The command above removes the "reflector" package. How come that its service is still enabled? Doesn't package removal clean up after itself properly?

Reply:
Ah! This is a custom added systemd unit for the cloud image, which triggered a systemd bug which I probably should report. Described in the description.